doc/readme.md

```diff
@@ -230,7 +230,7 @@ cordova {platform | platforms} [
 |     | --save                   | Updates the version specified in `config.xml` |
 |     | --nofetch                | Do not fetch the platform using `npm install` and do not store it into the apps `node_modules` directory |
 | list |                         | List all installed and available platforms |
-| check |                        | List platforms which can be updated by `cordova-cli platform update` |
+| check |                        | List platforms which can be updated from cordova-cli with the command `platform update` |
```

Comment:
What about "... which can be updated via `cordova-cli platform update`" ?

Reply:
I like the "via" part, but I'd like to keep the "With the command", so the people wont try to insert the full "cordova-cli platform update" in the console.